bindings/python/iota_sdk/types/output_data.py

```diff
@@ -11,6 +11,19 @@
 
 @dataclass
 class OutputData():
+    """Output data.
+
+    Attributes:
+        outputId (HexStr): the associated output ID
+        metadata (OutputMetadata): metadata associated with the output
+        output (Output): the output itself
+        isSpent (bool): whether the output is spent
+        address (Address): the address associated with the output
+        networkId (str): the network ID associated with the output
+        remainder (bool): whether the output is a remainder
+        chain (List[int], optional): array of chain state indexes 
```

Comment:
types

Reply:
yeah ... that went into `part 2` (see previous commit message :stuck_out_tongue_closed_eyes: )